### ISSUE TITLE
CUB200 training result issue

### ISSUE THREAD

**Issue by cccxyc** [jingyuanli001/RFR-Inpainting#38]
![image](https://user-images.githubusercontent.com/69033868/109407141-68da9c00-79b9-11eb-82ad-79aa0a3e1fea.png)
![image](https://user-images.githubusercontent.com/69033868/109407150-7bed6c00-79b9-11eb-9b9a-f1a8b4515277.png)
hi there I trained bird dataset for 800000 iters，yet the results remain blur. Does this have somthing todo with my mask  setting?
I opened mask with mode='RGB' and deleted the code        'masks = torch.cat([masks]*3, dim = 1)'

**Comment by jingyuanli001** [comment]
I would assume this is due to some issues in the input mask format. Opening the mask/image with different mode is dangerous as it changes the range of value, e.g. the mask image could either be within range 0~1 (gray scale) or range 0~255 (8 bit). This then influence the values of the output images. One rule-of-thumb is to ensure that the masks used in generating composed image in model.py have only 0s and 1s in it. Otherwise things could be wired.

**Comment by cccxyc** [comment]
will, i did set mask.jpg with only 0 and 1 pixel. the celeba dataset generates better results at samilar iters. i guess it depends on the complicity of trainiing images
